src/frontends/tensorflow/src/op/reverse.cpp

```diff
@@ -3,47 +3,126 @@
 //
 
 #include "op_table.hpp"
-#include "openvino/opsets/opset8.hpp"
+#include "openvino/opsets/opset9.hpp"
 
 using namespace std;
-using namespace ov::opset8;
+using namespace ov;
+using namespace ov::opset9;
 
 namespace ov {
 namespace frontend {
 namespace tensorflow {
 namespace op {
+shared_ptr<Node> compute_sequence_lengths(const Output<Node>& input_shape, int64_t batch_axis, int64_t seq_axis) {
+    auto batch_axis_const = make_shared<Constant>(element::i64, Shape{1}, batch_axis);
+    auto seq_axis_const = make_shared<Constant>(element::i64, Shape{1}, seq_axis);
```

Comment:
is it necessary to mix precision of Constants?
maybe it's better to have i32 or i64 only

Reply:
Fixed.